lms/templates/learner_dashboard/program_details_sidebar.underscore

```diff
@@ -8,14 +8,61 @@
     <% } %>
 </aside>
 <aside class="aside js-course-certificates"></aside>
+<% if (isSubscriptionEligible) { %>
+<aside class="aside js-subscription-info subscription-info">
+    <h2 class="divider-heading">
+        <%- gettext(
+            hasActiveTrial
+            ? 'Trial subscription'
+            : subscriptionState === 'active'
+            ? 'Active subscription'
+            : 'Inactive subscription'
+        ) %>
+    </h2>
+    <div class="sidebar-section">
+        <div class="subscription-section">
+            <p class="my-0">
+                <%= HtmlUtils.interpolateHtml(
+                    gettext((
+                        subscriptionState === 'active'
+                        ? 'View your receipts or modify your subscription'
+                        : subscriptionState === 'inactive'
+                        ? 'Restart your subscription for {subscriptionPrice}/month. Your payment history is still available'
+                        : 'If you had a subscription previously, your payment history is still available'
+                    ).concat(' on the {a_start}Orders and subscriptions{a_end} page')),
+                    {
+                        subscriptionPrice,
+                        a_start: HtmlUtils.HTML(`<a class="subscription-link" href="${manage_subscription_url}">`),
+                        a_end: HtmlUtils.HTML('</a>'),
+                    }
+                ) %>
+            </p>
+            <p class="my-0">
+                <%= HtmlUtils.interpolateHtml(
+                    gettext(
+                        'Need help? Check out the {a_start}Learner help centre{span_start}{icon}{span_end}{a_end} to troubleshoot issues or contact support '
+                    ),
+                    {
+                        a_start: HtmlUtils.HTML(`<a class="subscription-link" href="${subscriptions_learner_help_center_url}">`),
```

Comment:
According to the resolved comment above (https://github.com/openedx/edx-platform/pull/32060/files#r1182983156) if we're keeping the spelling of "centre", then this url should probably stick by that standard, no?

Reply:
Thanks for pointing that out, I have fixed the spelling to be "center" which is what the actual support page spells it as, I am just much more used to British English (so is my spell-checker)